online.hatsune_miku.tachidesk-vaadinui.json

```diff
@@ -0,0 +1,63 @@
+{
+  "app-id": "online.hatsune_miku.tachidesk-vaadinui",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.openjdk17"
+  ],
+  "command": "vaaui.sh",
+  "modules": [
+    {
+      "name": "openjdk",
+      "buildsystem": "simple",
+      "build-commands": [
+        "/usr/lib/sdk/openjdk17/install.sh"
+      ]
+    },
+    {
+      "name": "Tachidesk-VaadinUI",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p /app/bin",
+        "ls",
+        "mv Tachidesk-VaadinUI-*.*.*.jar /app/bin/Tachidesk-VaadinUI.jar",
+        "install tachidesk-vaadinui.sh /app/bin/vaaui.sh",
+        "chmod +x /app/bin/vaaui.sh",
+        "install -D -m0644 online.hatsune_miku.tachidesk-vaadinui.desktop /app/share/applications/online.hatsune_miku.tachidesk-vaadinui.desktop",
+        "install -D -m0644 online.hatsune_miku.tachidesk-vaadinui.png /app/share/icons/hicolor/128x128/apps/online.hatsune_miku.tachidesk-vaadinui.png",
+        "install -D -m0644 online.hatsune_miku.tachidesk-vaadinui.appdata.xml /app/share/metainfo/online.hatsune_miku.tachidesk-vaadinui.appdata.xml"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "url": "https://github.com/Suwayomi/Tachidesk-VaadinUI/releases/download/v1.0.3/Tachidesk-VaadinUI-1.0.3.jar",
+          "sha256": "f4ed7c0537be341f22443c49a0dd0c3afad001f1eb8adb08538963875e1b323d"
+        },
+        {
+          "type": "file",
+          "path": "tachidesk-vaadinui.sh"
+        },
+        {
+          "type": "file",
+          "path": "online.hatsune_miku.tachidesk-vaadinui.desktop"
+        },
+        {
+          "type": "file",
+          "path": "online.hatsune_miku.tachidesk-vaadinui.appdata.xml"
+        },
+        {
+          "type": "file",
+          "path": "online.hatsune_miku.tachidesk-vaadinui.png"
+        }
+      ]
+    }
+  ],
+  "finish-args": [
+    "--env=PATH=/app/jre/bin:/app/bin/",
+    "--share=network",
+    "--filesystem=home",
```

Comment:
why do you need home?

Reply:
I store most information in the home directory under Linux. The separate server process I rely on needs a path to put its information and for better or worse that's where I decided to put it on Linux distributions

Reply:
maybe you limit to that specific directory.

Reply:
> maybe you limit to that specific directory.

Done